spec/support/fixtures/openapi.json

```diff
@@ -799,12 +799,28 @@
           },
           "string": {
             "type": "string"
+          },
+          "nested": {
+            "$ref": "#/components/schemas/ObjectLookup"
           }
         },
         "required": [
           "unix"
         ]
       },
+      "ObjectLookup": {
```

Comment:
This ObjectLookup hasn't changed, it's just moved in the schema output because it gets generated earlier due to the tweak I made to `TimeLookupArgumentSet`.